backend/models/users.py

```diff
@@ -1,14 +1,16 @@
 """User models."""
+import uuid
+
 from models.base import BaseModel
 from sqlalchemy import Column, DateTime, ForeignKey, Integer, String, func
+from sqlalchemy.dialects.postgresql import UUID
 from sqlalchemy.orm import column_property, relationship
 
 
 class User(BaseModel):
     """Main User model."""
-
     __tablename__ = "users"
-    id = Column(Integer, primary_key=True)
```

Comment:
I think it would be OK to do it this way but having id as an Integer and the only PK on the table may be preferable since it is a little simpler.

Reply:
Yeah, I've thought about this, and I think my main reason for swapping it to use the UUID as the PK, was because that's how we'd plan on looking up the User most times anyway. With the BaseView, etc - we'd have to create additional endpoints or query string filters to search by `oidc_id` specifically. So just thought it easier to swap it out. 

But I do see where it could create some conflict.

Reply:
I will update this in the next PR